src/components/IntakeForm/Inputs/PeoplePicker.tsx

```diff
@@ -4,7 +4,7 @@ import {
   Label,
   NormalPeoplePicker,
 } from '@fluentui/react';
-import { usePeoplePicker } from 'components/Hooks/usePeoplePicker';
+import { usePeoplePicker } from 'components/Hooks/usePeoplePicker/usePeoplePicker';
```

Comment:
consider using 'components/Hooks' instead of 'components/Hooks/usePeoplePicker/usePeoplePicker'